src/App.js

```diff
@@ -70,11 +70,36 @@ function App() {
         if (user) {
           setAuthenticatedUser(user);
 
-          // localStorage.setItem("user", JSON.stringify(user));
+          localStorage.setItem("user", JSON.stringify(user));
```

Comment:
Your signup should be setting a `token` rather than a `user`.

Reply:
Check your controller to see if it is sending a token or a user.

Reply:
it's sending a token =) I'll change that